PortafolioMobile.html

```diff
@@ -130,15 +131,15 @@ <h3>"Teach yourself social networks" website</h3>
           </div>
 
           <div class="work-image">
-            <img src="images/portfolio/most-useful-websites.jpg" alt="image most-useful-websites.jpg" />
+            <img src="images/portfolio/SocialMediaLogos.jpg" alt="image SocialMediaLogos.jpg" />
           </div>
 
         </article>
       
         <article class="art_img_left">
 
           <div class="work-image">
-            <img src="images/portfolio/Website.png" alt="image Website.png" />
+            <img src="images/portfolio/TeachSeniors.png" alt="image TeachSeniors.png" />
```

Comment:
Hello @DarioAlessioR 
Geat job implementing the different features of this milestone in the work section the following are required:

- [ ] Please remove the different image `img tag` and implement it with a proper `background-image style`